include/threadClass.hpp

```diff
@@ -1,3 +1,28 @@
+/**
+ *  Schulich Delta OnBoard Media Control
+ *  Copyright (C) 2015 University of Calgary Solar Car Team
+ *
+ *  This file is part of the Schulich Delta OnBoard Media Control
+ *
+ *  The Schulich Delta OnBoard Media Control is free software: 
+ *  you can redistribute it and/or modify it under the terms 
+ *  of the GNU Affero General Public License as published by 
+ *  the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  The Schulich Delta OnBoard Media Control is distributed 
+ *  in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+ *  without even the implied warranty of MERCHANTABILITY or 
+ *  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero 
+ *  General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General 
+ *  Public License along with the Schulich Delta OnBoard Media 
+ *  Control.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  For further contact, email <software@calgarysolarcar.ca>
+ */
```

Comment:
So stackoverflow answers are under the Creative Commons license. I've asked Jonah to take a look at how that applies to us. In general, I think it would be better if we rewrote it anyways. 

It might be a good exercise for you to explain to Gordo, what this class class is supposed to do, teach him about virtual functions and threading etc, and he will rewrite the class. But this will be time extensive so it might not be worth our time. What do you think?